### ISSUE TITLE
Default role does not seem to work with Ansible 2.

### ISSUE THREAD

**Issue by nmische** [silpion/ansible-tomcat#22]
With Ansbile 2.0.0.2, the sample playbook seems to hang while downloading Tomcat. 

```
TASK [groover.tomcat : include] ************************************************
```

I have a very basic playbook:

```---
- hosts: all
  roles:
    - { role: groover.tomcat }
```
using the following `requirements.yml`:

```
---
- src: https://github.com/silpion/ansible-util/archive/1.1.0.tar.gz
  name: silpion.util

- name: groover.tomcat
  version: 1.3.0
```

**Comment by spul** [comment]
Right. Currently working on getting everything v2 ready and add some minor benefits.

**Comment by nmische** [comment]
Great, thanks!

**Comment by spul** [comment]
Fixed with [2.0.0][1].

[1]: https://github.com/silpion/ansible-tomcat/tree/2.0.0
